ckan/templates-midnight-blue/home/index.html

```diff
@@ -41,12 +40,24 @@
             {% snippet 'home/snippets/featured_group.html' %}
```

Comment:
The new design has the following sections:

Groups
Datasets
Organizations

I'd say that groups are the less used of the three so perhaps I'd suggest reordering to 

Datasets
Organizations
Groups

or 

Organizations
Datasets
Groups

@ckan/core any thoughts?

![Screenshot 2025-01-14 at 12-16-46 Welcome - CKAN](https://github.com/user-attachments/assets/09c45315-b352-406e-b6b2-abb41dd87246)